bundles/org.openhab.binding.daikin/src/main/java/org/openhab/binding/daikin/internal/handler/DaikinBaseHandler.java

```diff
@@ -201,7 +203,7 @@ private synchronized void poll() {
     }
 
     protected void updateTemperatureChannel(String channel, Optional<Double> maybeTemperature) {
-        updateState(channel, maybeTemperature.<State> map(t -> new QuantityType<>(new DecimalType(t), SIUnits.CELSIUS))
+        updateState(channel, maybeTemperature.<State>map(t -> new QuantityType<>(new DecimalType(t), SIUnits.CELSIUS))
```

Comment:
I don't know why you are passing in the decimal type into the quantity type. I'm pretty sure you don't need to do that.
```suggestion
        updateState(channel, maybeTemperature.<State>map(t -> new QuantityType<>(t, SIUnits.CELSIUS))
```

Reply:
Fixed.